src/vec.cpp

```diff
@@ -1090,86 +1090,20 @@ void grid_volume::find_best_split(int desired_chunks, int &best_split_point,
   }
 }
 
-grid_volume grid_volume::split_by_cost(int desired_chunks, int proc_num) const {
-  int best_split_point;
-  direction best_split_direction;
-  double left_effort_fraction;
-  find_best_split(desired_chunks, best_split_point, best_split_direction, left_effort_fraction);
-  return split_by_cost(desired_chunks, proc_num, best_split_point, best_split_direction,
-                       left_effort_fraction);
-}
-
-grid_volume grid_volume::split_by_cost(int desired_chunks, int proc_num, int best_split_point,
-                                       direction best_split_direction,
-                                       double left_effort_fraction) const {
-  if (desired_chunks == 1) return *this;
-
-  const int split_point = best_split_point;
-  const int num_in_split_dir = num_direction(best_split_direction);
-
-  const int num_low = (size_t)(left_effort_fraction * desired_chunks + 0.5);
-  // Revert to split() when cost method gives less grid points than chunks
```

Comment:
I'm not sure how important this check is?  Not sure if we need to put a version of this into `split_into_n` or something?

Reply:
That comes from the original `split_by_effort`, so I'm not sure how important it is either.